src/Monitor/Monitor/Alerts/NewAzureRmMetricAlertRuleV2CriteriaCommand.cs

```diff
@@ -25,12 +25,25 @@ namespace Microsoft.Azure.Commands.Insights.Alerts
     /// <summary>
     /// Create Metric Criteria
     /// </summary>
-    [Cmdlet("New", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "MetricAlertRuleV2Criteria", DefaultParameterSetName = StaticThresholdParameterSet), OutputType(typeof(PSMetricCriteria))]
+    [Cmdlet("New", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "MetricAlertRuleV2Criteria", DefaultParameterSetName = StaticThresholdParameterSet), OutputType(typeof(IPSMultiMetricCriteria))]
```

Comment:
Potential breaking change: Change of output type (unless the type contains all the properties the previous output type did